src/electron/automated-checks/components/automated-checks-command-bar.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import * as React from 'react';
+
+import { NamedFC } from 'common/react/named-fc';
+import { DeviceConnectActionCreator } from 'electron/flux/action-creator/device-connect-action-creator';
+import { automatedChecksCommandBar } from './automated-checks-command-bar.scss';
+
+export type AutomatedChecksCommandBarDeps = {
+    deviceConnectActionCreator: DeviceConnectActionCreator;
+};
+
+export interface AutomatedChecksCommandBarProps {
+    deps: AutomatedChecksCommandBarDeps;
+}
+
+export const AutomatedChecksCommandBar = NamedFC<AutomatedChecksCommandBarProps>(
+    'AutomatedChecksCommandBar',
+    (props: AutomatedChecksCommandBarProps) => {
+        const onClick = () => props.deps.deviceConnectActionCreator.resetConnection();
+        return (
+            <div className={automatedChecksCommandBar}>
+                <ActionButton
+                    className="button-rescan"
```

Comment:
I can't find styles for `button-rescan` style. If we're using this only for the sake of testing (like in `.find('.button-rescan')`) I'll suggest using an id or use the text property (like in this test `src\tests\unit\electron\device-disconnected-popup\device-disconnected-popup.test.tsx`)

Reply:
That is the only reason it has a className. Thanks for the tip--will update!

Reply:
Updated to use text